### DISCUSSION TITLE
Possible to download SQL tables?

### DISCUSSION THREAD

**Discussion by halpert3** [codewars/codewars.com#2410] [Q&A] [answered]
One of my tracks in Codewars is SQL. For most SQL kata, they come with given tables prepopulated with data. 

For other languages, I find it easier to troubleshoot potential solutions on my laptop locally, then copy and paste them to the Codewars website for the attempts.

For each SQL kata, is it possible to download the tables/data to my laptop, and if so, how do I do it?

**Comment by kazk** [comment, marked as answer]
No, it's usually dynamically generated in the tests written in Ruby.
